cve_bin_tool/checkers/perl.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python3
+"""
+CVE checker for Perl
+References:
+https://www.cvedetails.com/vulnerability-list/vendor_id-1885/Perl.html
+"""
+from ..util import regex_find
+
+def get_version(lines, filename):
+    regex = [r"Perl: ([5]+\.[0-9]+\.[0-9]+)"]
+    version_info = dict()
+    if filename[::-1].startswith(("perl")[::-1]):
```

Comment:
Why use filename[::-1] here?  I don't think you need or want to invert the string.  Was there another checker that has this construction that you copied from that maybe we should also fix?

Reply:
What this is doing in case you haven't opened up a python console to try it out:

```
>>> filename = "perl.blah.so"
>>> filename[::-1].startswith(("perl")[::-1])
False
>>> filename[::-1]
'os.halb.lrep'
```

Definitely not the right behaviour.